docs/src/pages/[platform]/connected-components/authenticator/customization/customization.i18n.react-native.mdx

```diff
@@ -21,7 +21,7 @@ The `Authenticator` ships with [translations](https://github.com/aws-amplify/amp
 These [translations](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts) can be customized using the [Amplify JS' `I18n`](https://docs.amplify.aws/lib/utilities/i18n/q/platform/js/) module:
 
 ```js
-import { I18n } from 'aws-amplify';
+import { I18n } from 'aws-amplify/utils';
```

Comment:
Should we leave a note about the previous path for v5 users?

Reply:
I added a Message above (and to the react-native page), do you think that works? 
<img width="942" alt="Screenshot 2024-02-07 at 3 47 54 PM" src="https://github.com/aws-amplify/amplify-ui/assets/376920/e9f7b07c-29de-4898-8d15-67eca93f94b7">